superset/connectors/sqla/models.py

```diff
@@ -71,12 +72,14 @@
 from sqlalchemy.sql.selectable import Alias, TableClause
 
 from superset import app, db, is_feature_enabled, security_manager
+from superset.columns.models import Column as NewColumn
 from superset.common.db_query_status import QueryStatus
 from superset.connectors.base.models import BaseColumn, BaseDatasource, BaseMetric
 from superset.connectors.sqla.utils import (
     get_physical_table_metadata,
     get_virtual_table_metadata,
 )
+from superset.datasets.models import Dataset as NewDataset
```

Comment:
Maybe alias this as something like `DatasetV2`? I get that it's "new and shiny" but it won't always be the case :).

Reply:
After this PR the plan is to get rid of the old models, so this will go away before we have v3.